src/WebApiToTypeScript/Config/Config.cs

```diff
@@ -12,6 +12,9 @@ public class Config
         public string EndpointsOutputDirectory { get; set; }
             = "Endpoints";
 
+        public bool EndpointsSupportCaching { get; set; }
+            = true;
```

Comment:
Since we now have folks using it, can we default to `false` for all new features please?